prow/config.yaml

```diff
@@ -1260,6 +1260,32 @@ postsubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  - name: maintenance-boskos-config-upload
+    agent: kubernetes
+    branches:
+    - master
```

Comment:
yeah (though we only have one branch in test-infra), we do the same thing for testgrid.